client/app/services/usage-graphs/_usage-graphs.sass

```diff
@@ -0,0 +1,22 @@
+.metric-title
+  font-size: 24px
+  text-align: left
+
+.metric-number
+  display: block
+  float: left
+  font-size: 20px
+  text-align: left
+
+.metric-details-container
+  height: 30px
+
+.metric-details
+  display: block
+  float: left
+  padding-left: 10px
+  padding-top: 3px
+
+  p
```

Comment:
This is indented to where its only p's contained inside of elements of the .metric-details class.   Its scoped so its not a global p change.  Just for paragraphs inside .metric-details